riscv.c

```diff
@@ -874,15 +1453,24 @@ void rv_step(struct riscv_t *rv, int32_t cycles)
             // dispatch this opcode
             TABLE_TYPE op = jump_table[index];
             assert(op);
+            rv->inst_len = INST_32;
             if (!op(rv, inst))
                 break;
 
             // increment the cycles csr
             rv->csr_cycle++;
         } else {
             // TODO: compressed instruction
```

Comment:
Remove "TODO" here since you provided the preliminary compressed instruction support.